src/terraform/commands.go

```diff
@@ -10,6 +10,9 @@ import (
 
 // RunInit runs the 'terraform init' command and force copies the remote state
 func RunInit(deployConfig types.DeployConfig) error {
+	if !util.DoesFileExist(deployConfig.TerraformFile) {
+		return errors.New("terraform initialization attempted in directory with no 'main.tf' file. Please make sure 'exo deploy build' has been run")
+	}
```

Comment:
Oh I see, so only when `main.tf` doesn't exist does it run `exo deploy build` automatically, otherwise it's up to the user whether or not they overwrite the existing file.